src/services/signalr.configuration.ts

```diff
@@ -19,13 +19,17 @@ export class SignalRConfiguration {
      /** Allows withCredentials. This flag can be used to suppport CORS */
     public withCredentials: boolean;
 
+    /**Allows you to specify transport. You can specify a fallback order if you wan't to try specific transports in order. */
+    public transport?: any;
```

Comment:
This is either a string or array of strings. I can change it to array of TransportConnection objects, and change it to string or array of strings before passing it to connect function. This way user wont be able to pass invalid parameters.